packages/neuron-wallet/src/block-sync-renderer/sync/check-and-save/tx.ts

```diff
@@ -49,25 +52,24 @@ export default class CheckTx {
     return false
   }
 
-  public filterOutputs = (lockHashes: string[]) => {
-    const cells: Cell[] = this.tx.outputs!.map((output, index) => {
+  public selectOutputsOfWallets = (lockHashes: string[]) => {
+    const outputs: Output[] = this.tx.outputs!.map((output, index) => {
       const checkOutput = new CheckOutput(output)
       const result = checkOutput.checkLockHash(lockHashes)
       if (result) {
         if (output.type) {
-          this.tx.outputs![index].typeHash = LockUtils.computeScriptHash(output.type)
           if (output.typeHash === this.daoTypeHash) {
-            this.tx.outputs![index].daoData = this.tx.outputsData![index]
+            this.tx.outputs![index].setDaoData(this.tx.outputsData![index])
           }
         }
         return output
       }
       return false
-    }).filter(cell => !!cell) as Cell[]
-    return cells
+    }).filter(output => !!output) as Output[]
+    return outputs
   }
 
-  public filterInputs = async (lockHashes: string[]): Promise<string[]> => {
+  public selectInputsOfWallets = async (lockHashes: string[]): Promise<string[]> => {
```

Comment:
Does it actually select addresses instead of inputs? If so it should be called `selectAddressesOfWallets`.

Reply:
Maybe `selectInputAddressesOfWallets` ?

Reply:
That sounds good to me.

Reply:
Renamed